nomos-blend/core/src/message_blend/mod.rs

```diff
@@ -43,33 +39,26 @@ where
 /// - Unwraps incoming messages received from network using
 ///   [`CryptographicProcessor`]
 /// - Pushes unwrapped messages to [`TemporalProcessor`]
-pub struct MessageBlendStream<S, NodeId, Rng, M, Scheduler>
-where
-    M: BlendMessage,
-{
+pub struct MessageBlendStream<S, NodeId, Rng, Scheduler> {
```

Comment:
Single-letter `S` generic?

Reply:
I mean this will soon go anyway in my upcoming refactor, so not really important.